chore(learn): Move skills tests into subdir (#39262)
mdn/content: files/en-us/learn_web_development/extensions/advanced_javascript_objects/test_your_skills/index.md

---
title: "Test your skills: Advanced JavaScript objects"
short-title: Test your skills
slug: Learn_web_development/Extensions/Advanced_JavaScript_objects/Test_your_skills
page-type: learn-module-assessment
sidebar: learnsidebar
---

> This page lists Advanced JavaScript objects so you can verify you've understood the content in this module.

Reviewer: or?
```suggestion
This page lists tests to help you assess your understanding of Advanced JavaScript objects.
```

> ## Test your Advanced JavaScript objects skills by topic

Reviewer: how about this:
```suggestion
## Test your skills in advanced JavaScript objects by topic
```
Do we need "by topic"? Even for future proofing, if I imagine us adding more tests here, probably "by topic" is redundant
